packages/velaux-ui/src/pages/Cluster/components/AddClusterDialog/index.tsx

```diff
@@ -55,27 +55,28 @@ class AddClusterDialog extends React.Component<Props, State> {
       });
     }
   };
-
+  
   onClose = () => {
     this.props.onClose();
     this.resetField();
   };
 
   onOk = () => {
     const { editMode, cluster } = this.state;
+    const { editClusterName } = this.props;
```

Comment:
What's the need for this code change?

Reply:
Thank you for pointing this out! I'll go ahead and remove it to clean up the code.

Reply:
> What's the need for this code change?

Thank you for pointing this out! I'll go ahead and remove it to clean up the code.